js/views/pauseScreen.js

```diff
@@ -54,6 +61,18 @@ var PauseScreen = React.createClass({
     this.props.handleVrPlayerClick();
   },
 
+  /**
+   * remove the button on pause screen for correct checking mouse movement
+   */
+  hideVrPauseButton: function() {
+    if (this.props.controller.videoVr) {
+      var pauseButton = document.getElementById('oo-pause-button');
+      setTimeout(function() {
+        pauseButton.style.display="none";
+      }, 1000);
```

Comment:
Why do we need a timeout? What is the significance of the value 1000?

Reply:
1000ms is time for the button animation in styles.